lib/cli/writer.js

```diff
@@ -67,6 +68,13 @@ exports.writeImage = (imagePath, drive, options, onProgress) => {
     return fs.openAsync(drive.raw, 'rs+');
   }).then((driveFileDescriptor) => {
     return imageStream.getFromFilePath(imagePath).then((image) => {
+      if (!image.size.final.estimation && image.size.final.value > drive.size) {
+        throw errors.createUserError(
+          'The image you selected is too big for this drive',
+          'Plug a bigger drive and try again'
```

Comment:
We should refactor the whole messages thing (I'll making a note to myself), given that the messages in `lib/shared/messages.js` contain the title and description mixed in a single string, where `errors.createUserError` requires them as separate arguments. 

I'll refactor in another PR.